cmd/kpod/common.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	is "github.com/containers/image/storage"
+	"github.com/containers/storage"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+var needToShutdownStore = false
+
+func getStore(c *cli.Context) (storage.Store, error) {
+	options := storage.DefaultStoreOptions
+	if c.GlobalIsSet("root") || c.GlobalIsSet("runroot") {
```

Comment:
These two are specified in `crio.conf` - we probably want to share them with CRI-O, so loading a CRI-O config and using the values from that seems like a good idea